pages/tips.md

```diff
@@ -24,6 +24,6 @@ _If you want to contribute, don't hesitate to send us a Pull Request with your t
 7. [Configuring Email with - Gmail and more]({{ site.url }}/tips/011_tip_configuring_email_in_jhipster.html)
 8. [Add a new Spring Social connector]({{ site.url }}/tips/012_tip_add_new_spring_social_connector.html)
 9. [Speed up the generator-jhipster]({{ site.url }}/tips/013_tip_speed_up_generator.html)
-10. [Using MariaDB]({{ site.url }}/tips/014_tip_using_mariadb.html)
```

Comment:
Yes I can add for gradle I do it all the time. Even better ill ask my
junior to do it so he can start contributing it as well :)

Thanks & regards,
Deepu

On 19 Aug 2016 23:54, "Pascal Grimaud" <notifications@github.com> wrote:

> In pages/tips.md
> <https://github.com/jhipster/jhipster.github.io/pull/297#discussion_r75529376>
> :
>
> > @@ -24,6 +24,6 @@ _If you want to contribute, don't hesitate to send us a Pull Request with your t
> >  7. [Configuring Email with - Gmail and more]({{ site.url }}/tips/011_tip_configuring_email_in_jhipster.html)
> >  8. [Add a new Spring Social connector]({{ site.url }}/tips/012_tip_add_new_spring_social_connector.html)
> >  9. [Speed up the generator-jhipster]({{ site.url }}/tips/013_tip_speed_up_generator.html)
> > -10. [Using MariaDB]({{ site.url }}/tips/014_tip_using_mariadb.html)
>
> No, just deleted the link, not the page
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jhipster/jhipster.github.io/pull/297/files/a56c0851957493c0c7294c76e10bd6f7d32f49f3#r75529376>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABDlF7sLTZI8iEyksBwLwhEV9SS1nizFks5qhfTSgaJpZM4Jonhi>
> .
>